Core/Prognetics.Saga.Parsers.DependencyInjection/JsonParserExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Prognetics.Saga.Core.Abstract;
+using Prognetics.Saga.Parsers.Core.Model;
+using System.Reflection;
+
+namespace Prognetics.Saga.Parsers.DependencyInjection
+{
+    public static class JsonParserExtensions
+    {
+        public static void RegisterJsonParsers(this IServiceCollection services,
```

Comment:
I think RegisterParsers would be more accurate

Reply:
Done.